webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -112,12 +121,14 @@ PageLayout.propTypes = {
       })
     ),
   }),
-  toolbarButtons: PropTypes.node,
   toastNotifications: PropTypes.string,
   onSearch: PropTypes.func,
   onBookmarkClick: PropTypes.func,
   searchQuery: PropTypes.string,
   beforeToolbarComponent: PropTypes.node,
+  isExport: PropTypes.bool,
+  isDocs: PropTypes.bool,
```

Comment:
IMO the `isExport` and `isDocs` are redundant.
As a consumer, I would prefer to pass `exportUrl` and `docsUrl` so if I pass one of those props it would render the button and if not it won't render it.

Reply:
this will cause rewriting of the generating code for exportUrl and I think we should generate the export url as it is always the same

Reply:
I wouldn't say rewriting, move the logic to a helper file and import it